cypress/support/fragments/data_import/logs/logs.js

```diff
@@ -113,6 +113,14 @@ export default {
       .and('include', 'underline');
   },
 
+  verifyNoFileNameFileAbsent: () => {
```

Comment:
verifyNoFileNameLogAbsent or verifyNoFileNameTitleAbsent

Reply:
Fixed, thank you!